src/routes/authRoutes.js

```diff
@@ -37,26 +40,26 @@ router.get('/callback', (req, res) => {
 });
 router.get(
 
-  "/auth/google",
-  passport.authenticate("google", { scope: ["email", "profile"] })
+  '/auth/google',
+  passport.authenticate('google', { scope: ['email', 'profile'] }),
 );
 router.get(
-  "/auth/google/callback",
-  passport.authenticate("google", {
+  '/auth/google/callback',
+  passport.authenticate('google', {
     session: false,
-    failureRedirect: "/",
+    failureRedirect: '/',
   }),
-  googleAuthHandler
+  googleAuthHandler,
 );
 
```

Comment:
an this line also here
**router.post('/vendor',isAdmin, vendor);**

Reply:
Done